client/blocks/legal-updates-banner/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { useEffect } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestLegalData } from 'state/legal/actions';
+
+const LegalUpdateBanner = ( { fetchLegalData } ) => {
```

Comment:
I think putting this inline with the component is fine for our one off banner, since we're likely to delete this later 👍. For folks reading our other query components live in `client/components/data`